### PR TITLE
dead link

### PR DESCRIPTION
**Pull request by stephanlindauer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,6 @@ Calico networking for Kubernetes can now be found in the [Calico CNI repository]
 
 We recommend migrating to use the Calico CNI plugin for Kubernetes deployments.  Documentation and guides for using the
 CNI plugin with Kubernetes can be found in the [calico-containers
-repository](https://github.com/projectcalico/calico-containers/blob/master/docs/cni/kubernetes/README.md).
+repository](https://github.com/projectcalico/calico-containers/).
```
Comment (caseydavenport) [inline README.md]: The Calico documentation has actually moved to http://docs.projectcalico.org, so it's a super dead link :)

**Comment by stephanlindauer** [comment]
done

File: README.md [hunk 2]
```diff
@@ -9,6 +9,6 @@ Calico networking for Kubernetes can now be found in the [Calico CNI repository]
 
 We recommend migrating to use the Calico CNI plugin for Kubernetes deployments.  Documentation and guides for using the
 CNI plugin with Kubernetes can be found in the [calico-containers
-repository](https://github.com/projectcalico/calico-containers/blob/master/docs/cni/kubernetes/README.md).
```
Comment (caseydavenport) [inline README.md]: Could you also update the link text from "calico-containers repository" to "the Calico docs site" :)

Comment (stephanlindauer) [inline README.md]: done

**Comment by caseydavenport** [comment]
Thanks!

**Pull request merged**